frontend/src/components/common/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  icon: string;
+  onClick: () => void;
+};
+
+const IconButton: FC<Props> = ({ icon, onClick }) => {
+  return (
+    <button className={styles.button} onClick={onClick}>
+      <img src={icon} alt="Action" />
```

Comment:
```suggestion
      <img src={icon} alt={name} />
```
pass name through props